lib/builderator/util.rb

```diff
@@ -79,6 +79,48 @@ def asg(region = Config.aws.region)
         clients["asg-#{region}"] ||= Aws::AutoScaling::Client.new(:region => region)
       end
 
+      def remove_security_group(region = Config.aws.region, group_id = nil)
+        if region.nil?
+          puts "  Dry-run; skipping delete of group_id #{group_id}"
+          return
+        end
+        ec2 = ec2(region)
+        resp = ec2.delete_security_group(group_id: group_id)
+        puts "  Deleted SecurityGroup #{group_id}"
+      end
+
+      def get_security_group_id(region = Config.aws.region)
+        group_id = nil
+        if region.nil?
+          group_id = 'sg-DRYRUNSG'
+          puts "  Dry-run; skipping create and returning #{group_id}"
+          return group_id
+        end
+        ec2 = ec2(region)
+        group = nil
+        require 'open-uri'
+        external_ip = open('http://checkip.amazonaws.com').read.strip
```

Comment:
I don't think this will be an issue, and it is, we can solve it when it becomes one.  The standard is to have a single egress IP from any one point in an internal network.  If it becomes a problem, then I can figure out a way to test and then solve it.

(cc @udangel-r7 and @kgawt-r7)